config/pull_policy.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"github.com/pkg/errors"
+)
+
+// PullPolicy defines a policy for how to manage images
+type PullPolicy int
+
+const (
+	// PullAlways images, even if they are present
+	PullAlways PullPolicy = iota
```

Comment:
`iota` is useful for defining enums. For more on iotas, see https://github.com/golang/go/wiki/Iota or  https://splice.com/blog/iota-elegant-constants-golang/

On the other hand, I also debated just defining these as strings, which would've obviated the need for a secondary `String()` method. If people would rather change it to a string type to make it simpler, I'm happy to do that.